bedrock/firefox/templates/firefox/compare/index.html

```diff
@@ -0,0 +1,409 @@
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('Six of the best browsers in direct comparison') }}{% endblock %}
+
+{% block page_desc %}
+{{ _('Find out how Firefox, Chrome, Edge, Safari, Opera and Internet Explorer differ in terms of privacy, utility and portability.') }}
+{% endblock %}
+
+{% block page_css %}
+{{ css_bundle('compare') }}
+{% endblock %}
+
+{% block content %}
+<section class="mzp-c-hero mzp-has-image compare-hero">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+
+      <p class="compare-hero-subtitle">{{ _('Privacy. Utility. Portability.') }}</p>
+
+      <h1 class="mzp-c-hero-title">{{ _('Six of the best browsers in direct comparison') }}</h1>
+
+      <div class="mzp-c-hero-desc">
+        <p>{{ _('Looking for a better browser? We’ll compare Firefox with Chrome, Edge, Safari, Opera and Internet Explorer to help you make your decision.') }}</p>
+      </div>
+
+      {{ download_firefox(dom_id='compare-hero-download', download_location='primary cta') }}
+
+    </div>
+  </div>
+  <div class="mzp-c-hero-image">
+      <img src="{{ static('img/firefox/compare/compare-browser-windows.png') }}" alt="" height="312" width="466">
+  </div>
+</section>
+
+<div class="mzp-l-content l-compare-content">
+  <article class="mzp-c-article">
+    <section class="compare-section-summary">
+
+      <div class="compare-section-summary-intro">
+        <p>
+          {{ _('A great internet browser should have the functionality you need, portability across devices, and the privacy you deserve.') }}
+        </p>
+
+        <div class="compare-section-summary-image">
+            <img src="{{ static('img/firefox/compare/compare-hero.svg') }}" alt="" height="403" width="532">
+        </div>
+      </div>
+
+      <div class="compare-section-summary-contents">
+        <p>
+          {% trans %}
+          Since your browser is your gateway to the internet, speed, security, privacy and utility are paramount. In recent years, Google Chrome has been the browser of choice for many. But at a time when online ads seem to follow us everywhere and data breaches are a fixture of news headlines, a lot of people are starting to demand more privacy and respect from their browser.
+          {% endtrans %}
+        </p>
+
+        <p>
+          {% trans %}
+          So is your browser the best one for what you do online? The right browser can make a big difference in how you experience the web. So, without further ado, let’s compare Google Chrome, Firefox, Safari, Opera, Microsoft Internet Explorer and Edge — and see which best suits your needs.
+          {% endtrans %}
+        </p>
+      </div>
+    </section>
+
+    <section class="compare-section-privacy">
+      <header class="compare-section-header">
+        <h2>
+          {# L10n: The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. #}
+          {{ _('<strong>Privacy:</strong>') }}
```

Comment:
I wonder if this could get tricky for l10n... since it's a one word headline we could just put the `strong` around the whole string BUT I suspect some languages will translate this as more than one word, and the zap needs to be on a single word (since it can't handle text wrapping). I'm not sure how best to handle this. 

I think maybe we should keep the `strong` as part of the string and expand on the l10n note to encourage localizers to choose a single word if possible, or if the best translation is more than one word, they should choose one word to emphasize. The same goes for all the one-word headlines on this page.

Reply:
Good point - I've clarified the l10n notes in 
9806cb2.